packages/perseus/src/widgets/label-image/marker.tsx

```diff
@@ -102,90 +113,86 @@ export default class Marker extends React.Component<Props> {
             ];
         }
 
-        if (isSelected && selected && selected.length === 0) {
-            innerIcon = (
-                <Icon icon={iconChevronDown} size={8} color={Color.white} />
-            );
+        if (isSelected && selected) {
+            innerIcon = iconChevronDown;
         }
 
         return (
             <View
                 style={[styles.markerIcon, iconStyles]}
                 ref={(node) => (this._icon = node)}
+                onClick={() => {
+                    this.props.analytics?.onAnalyticsEvent({
+                        type: "perseus:label-image:marker-interacted-with",
+                        payload: null,
+                    });
+                }}
             >
-                {innerIcon}
+                <Icon
```

Comment:
For some unknown reason, this was not clickable when rendered conditionally. @SonicScrewdriver can attest!

Could be a weird Perseus Icon thing. Decided to hold off switching to WB.